test/unit/associations/belongs-to-many.test.js

```diff
@@ -11,7 +11,7 @@ const BelongsTo = require(__dirname + '/../../../lib/associations/belongs-to');
 const HasMany = require(__dirname + '/../../../lib/associations/has-many');
 const HasOne = require(__dirname + '/../../../lib/associations/has-one');
 const current = Support.sequelize;
-const Promise = current.Promise;
+var Promise = current.Promise;
```

Comment:
I honestly don't entirely understand why this qualifies as a redefinition... I suppose Promise exists as a builtin, but that builtin was not defined with `const`